Nexus-Mods/NexusMods.App
# collection-cards

Rough Draft of Collection Cards. The functionality is mostly here, but the styling needs some work. I'd done about all I can do as far as styling, and it's likely a waste of time to have me flounder on this much more. 

* There's a new page called `Collections (WIP)` that will eventually be rolled into the Library. For now it's easy enough to display it as a separate page so we don't have to feature flag it
* `GetOrAddCollectionMetadata` now makes a single GQL call and loads and saves off both the collection image and the Users' avatar image. 
* As collections are recognized we show the most recent revision in the Collections page, along with all the metadata
* `GetOrAddCollectionMetadata` is idempotent on data that has not changed, so we can update that later to re-query the data as needed.
* `0/12` mod counts are not yet implemented

To Test:
1. Manage a game
2. Go to the loadout and open the `Collections (WIP)` page
3. Go to NexusMods and download a small collection
4. The collection should show up in the new page

Yes it looks like trash...but the structure of the UI components should be correct, and we can change the styling. 

![Screenshot 2024-09-18 174807](https://github.com/user-attachments/assets/64403265-ff93-43d0-9284-0849f69212c1)
